README.md

```diff
@@ -88,7 +101,405 @@ workflow wf {
 }
 ```
 
-Architecture
-------------
+# Getting Started with WDL
+
+We'l use Cromwell to run these examples but you can use any WDL engine of your choice. If you don't already have a reference to the Cromwell JAR file, one can be [downloaded](https://github.com/broadinstitute/cromwell/releases) or [built manually](#building)
+
```

Comment:
The `#building` link looks like C/P from the Cromwell README and doesn't resolve in this README.